tests/foreman/cli/test_import.py

```diff
@@ -111,6 +112,8 @@ def update_csv_values(csv_file, new_data=None, dirname=None):
                     record.get('org_id') == change['key_id']):
                 record.update(change)
                 del record['key_id']
+                if(record.get('org_id')):
```

Comment:
@Ichimonji10 : 
yes, overwriting the key values is the actual purpose of the method.

The problem with Sat5 export files is, that some of them use `org_id` and the others use `organization_id` for referring to the same thing. This allows me to use `organization_id` for all as this statement should take care of it for me (It should literally remove the `organization_id` and assign it to the `org_id` in case `org_id` is used)